src/main/java/blackjack/domain/user/Status.java

```diff
@@ -0,0 +1,36 @@
+package blackjack.domain.user;
+
+import java.util.Arrays;
+import java.util.function.IntPredicate;
+
+public enum Status {
+    PLAYING(true, i -> i < Constants.BLACKJACK),
+    BLACKJACK(false, i -> i == Constants.BLACKJACK),
+    BURST(false, i -> Constants.BLACKJACK < i),
+    STOP(false, i -> false);
+
+    private static class Constants {
+        public static final String NO_MATCH_CONDITION_ERROR_MSG = "점수 조건에 맞는 Status가 없습니다.";
+        public static final int BLACKJACK = 21;
+    }
```

Comment:
Constants 까지 만들필요가 있을까 싶어요 :) 
```suggestion
    PLAYING(true, i -> i < Status.BLACKJACK_SCORE),
    BLACKJACK(false, i -> i == Status.BLACKJACK_SCORE),
    BURST(false, i -> Status.BLACKJACK_SCORE < i),
    STOP(false, i -> false);
    
    public static final String NO_MATCH_CONDITION_ERROR_MSG = "점수 조건에 맞는 Status가 없습니다.";
    public static final int BLACKJACK_SCORE = 21;
```